test/inductor/test_aot_inductor.py

```diff
@@ -63,9 +63,14 @@ def load(
             constraints=constraints,
         )
 
+        launcher = aot_inductor_launcher
+        is_cpu = any(x.device.type == "cpu" for x in example_inputs)
```

Comment:
Since the current code either puts all the weights on GPU or CPU, I don't think there's really a "correct" predicate here.  Maybe it would be better to look through the model weights :shrug: